.travis.yml

```diff
@@ -70,10 +70,8 @@ matrix:
       env: RUN=rspec
 
     - rvm: jruby-9.2.4.1
-      env: RUN=rspec REINSTALL_BUNDLER=true
```

Comment:
You should just remove `REINSTALL_BUNDLER=true` here, that's why the failures on jruby popped up. That doesn't explain the error, but explains why we didn't have it before.

Reply:
Ooops, nice catch :flushed:

Reply:
fixed